pp_sys.c

```diff
@@ -3905,9 +3905,9 @@ PP_wrapped(pp_chdir, MAXARG, 0)
     dSP; dTARGET;
     const char *tmps = NULL;
     GV *gv = NULL;
+    SV * const sv = MAXARG == 1 ? POPs : NULL;
 
-    if( MAXARG == 1 ) {
-        SV * const sv = POPs;
+    if (sv) {
```

Comment:
It took me a while to figure this out, perhaps it could use a comment about pp_coreargs pushing NULL (pp_index has one).

I had thought there was some discussion a while back about not allowing NULL on the stack, but I couldn't find it, and coreargs does it.

Reply:
Comment added.